dotcom-rendering/src/web/bork/fcp.ts

```diff
@@ -0,0 +1,19 @@
+export const fcp = (): void => {
```

Comment:
It might be good to have a little description, possibly [pointing to the CLS definition](https://web.dev/cls/).